include/fem-dem/cfd_dem_coupling.h

```diff
@@ -319,6 +319,7 @@ class CFDDEMSolver : public GLSVANSSolver<dim>
   DoFHandler<dim> background_dh;
   PVDHandler      grid_pvdhandler;
   PVDHandler      particles_pvdhandler;
+  TableHandler    particle_velocity_table;
```

Comment:
ifyou are going to print all the particle velocity, will not print everything related to the particles? Shahab has already made a post-processor for this. it's called print xyz or something like this and it is already used in the DEM. No need to make a new one :)

Reply:
I will remove it. It was just for one single case